docs/using.md

```diff
@@ -34,67 +35,76 @@ This section contains information about basic virtual machine and OpenShift mana
 <a name="starting-openshift"></a>
 ### Starting OpenShift
 
-The [minishift start](./minishift_start.md) command is used to start your OpenShift instance.
+The [`minishift start`](./minishift_start.md) command is used to start your OpenShift instance.
 This command creates and configures a virtual machine that runs a single-node OpenShift instance.
 
 <a name="stopping-openshift"></a>
 ### Stopping OpenShift
 
-The [minishift stop](./minishift_stop.md) command is used to stop your OpenShift instance.
+The [`minishift stop`](./minishift_stop.md) command is used to stop your OpenShift instance.
 This command shuts down the Minishift virtual machine, but preserves the cluster state.
 Starting Minishift again will restore the cluster, allowing you to continue work from where you left-off.
 
 <a name="deleting-openshift"></a>
 ### Deleting OpenShift
 
-The [minishift delete](./minishift_delete.md) command is used to delete the OpenShift instance.
+The [`minishift delete`](./minishift_delete.md) command is used to delete the OpenShift instance.
 This command shuts down and deletes the Minishift virtual machine. No data or state is preserved.
 
+<a name="controlling-runtime-behavior-of-minishift"></a>
+## Controlling runtime behavior of Minishift  
+
+The runtime behavior of Minishift can be controlled through flags, environment variables, and persistent configuration options, as discussed in the following sections.
+
+The following precedence order is applied to control the behavior of Minishift. Each item in the following list takes precedence over
+the item below it:
+
+1. Use a command line flag as specified in the [Flags](flags) section.
+1. Set environment variable as described in the [Environment variables](#environment-variables) section.
+1. Use persistent configuration option as described in the [Persistent configuration](#persistent-configuration) section.
+1. Set the default value as defined by Minishift.
+
+
+<a name="flags"></a>
+### Flags
+
+You can use command line flags with Minishift to specify options and direct its behavior. This has the highest precedence. Some of the commonly used command line flags are `cpus`, `memory` or `vm-driver`, for a full set of supported flags refer to the Synopsis section of the
+[`minishift config`](./minishift_config.md) sub-command.
```

Comment:
"Some of the commonly used command line flags are `cpus`, `memory` or `vm-driver`" => "Some of the commonly used command line flags of the start command are for example `cpus`, `memory` and`vm-driver"`"

Flags are not just for the start command, pretty much all commands have flags/options.